manifests/mod/security.pp

```diff
@@ -128,6 +128,8 @@
     }
   }
 
-  unless $::operatingsystem == 'SLES' { apache::security::rule_link { $activated_rules: } }
+  # the rule setup on ubuntu 18.04 is completely different,
+  # the default rules used within this module aren't available
+  unless $::operatingsystem == 'SLES' or $facts['os']['release']['major'] == '18.04' { apache::security::rule_link { $activated_rules: } }
```

Comment:
Might be a bit more readable to put the body on a new line. It's very long.

Reply:
fixed